internal/pgd/module.go

```diff
@@ -316,21 +329,17 @@ func (m *Module) applyKeyFuncs(f *jen.File, in pgs.File) error {
 	return nil
 }
 
-func generateKeyStringer(msg pgs.Message, stmts []jen.Code, ck *dynamopb.Key, stringBuffer string) []jen.Code {
+func generateKeyStringer(msg pgs.Message, stmts []jen.Code, fields []string, stringBuffer string) []jen.Code {
 	stmts = append(stmts, jen.Id(stringBuffer).Dot("Reset").Call())
 
-	sep := ck.Separator
-	if sep == "" {
-		sep = ":"
-	}
+	sep := ":"
+	prefix := msg.Name().LowerSnakeCase().String()
```

Comment:
dont GSIs and the PK need a different initial prefix?  could just be `${prefix}_gsi1` i think for the gsi1 - its just to separate out the data from conflicts?

Reply:
gsi's are in separate tables no? Do they conflict?

Reply:
Based on my reading of

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/SecondaryIndexes.html

It doesn't seem necessary, but I'll test it.